src/main/java/com/github/nkzawa/engineio/client/Socket.java

```diff
@@ -197,6 +200,7 @@ public Socket(Options opts) {
                 opts.transports : new String[]{Polling.NAME, WebSocket.NAME}));
         this.policyPort = opts.policyPort != 0 ? opts.policyPort : 843;
         this.rememberUpgrade = opts.rememberUpgrade;
+        this.okHttpClient = opts.okHttpClient;
```

Comment:
I think we should add `hostnameVerifier` option instead of adding `OkHttpClient` directly.